mods/player_api/api.lua

```diff
@@ -50,9 +50,12 @@ end
 
 -- Player stats and animations
 -- model, textures, animation
-local players = {}
+player_api.players = {}
```

Comment:
Why expose this? You can use `get_animation` to get entries from this table already, and you shouldn't be able to write to it directly; you should be using `set_[model|textures|animation]` for that instead.

Reply:
Whoops, I missed that